acpi/src/platform/mod.rs

```diff
@@ -37,10 +37,25 @@ pub struct Processor {
     pub is_ap: bool,
 }
 
-pub struct ProcessorInfo {
+pub struct ProcessorInfo<'a, A>
+where
+    A: core::alloc::Allocator,
+{
     pub boot_processor: Processor,
     /// Application processors should be brought up in the order they're defined in this list.
-    pub application_processors: Vec<Processor>,
+    pub application_processors: crate::ManagedSlice<'a, Processor, A>,
```

Comment:
Applies everywhere: ig this is why you've re-exported `ManagedSlice` from the root? I'd prefer to just `use` it in the file and have `ManagedSlice` everywhere

Reply:
In this instance, it allows us to avoid having to do something like:
```
#[cfg(feature = "allocator_api")]
use crate::managed_slice::ManagedSlice;
```

which is kind of verbose when done in every call location. Exporting it from the root allows easy-to-read direct imports, which allows us to not need the above code block.

Reply:
Sure